helm/cluster-aws/values.schema.json

```
@@ -583,6 +583,11 @@
                         }
                     }
                 },
+                "replicas": {
+                    "type": "integer",
+                    "title": "Number of control plane instances",
+                    "default": 3
+                },
```

Let's set a constraint, too

```suggestion
                    "default": 3,
                    "minimum": 1
                },
```